examples/microprofile/idcs/src/main/java/io/helidon/examples/microprofile/security/idcs/IdcsMain.java

```diff
@@ -35,13 +34,16 @@ private IdcsMain() {
      * @throws IOException when logging configuration fails
      */
     public static void main(String[] args) throws IOException {
-        LogManager.getLogManager().readConfiguration(IdcsMain.class.getResourceAsStream("/logging.properties"));
         Server.create().start();
 
         System.out.println("Endpoints:");
         System.out.println("Login");
         System.out.println("  http://localhost:7987/rest/login");
         System.out.println("Full security with scopes and roles (see IdcsResource.java)");
         System.out.println("  http://localhost:7987/rest/scopes");
+        System.out.println("A protected reactive service (see application.yaml - security.web-server)");
+        System.out.println("  http://localhost:7987/reactive");
+        System.out.println("A protected static resource (see application.yaml - security.web-server");
+        System.out.println("  http://localhost:7987/web/resource.html");
```

Comment:
shouldn't the string be rendered from the frontend URI in application.yaml?

Reply:
That is a possibility, though these printouts are really for the sake of the example. In a real application sitting in a container, such messages are useless. So I expect them to be removed.